build/terraform/providers/aws/main.tf

```diff
@@ -70,3 +84,53 @@ module "eks" {
   cluster_version  = var.cluster_version
   write_kubeconfig = true
 }
+
+resource "aws_iam_policy" "policy" {
+  name   = var.policy_name
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:AttachVolume",
+        "ec2:CreateSnapshot",
+        "ec2:CreateTags",
+        "ec2:CreateVolume",
+        "ec2:DeleteSnapshot",
+        "ec2:DeleteTags",
+        "ec2:DeleteVolume",
+        "ec2:DescribeInstances",
+        "ec2:DescribeSnapshots",
+        "ec2:DescribeTags",
+        "ec2:DescribeVolumes",
+        "ec2:DetachVolume"
+      ],
+      "Resource": "*"
```

Comment:
We could probably lock this down to the actual cluster resource we've created... but for now it'll be ok.